docs/maintain/maintain-guides-how-to-nominate-polkadot.md

```diff
@@ -60,6 +60,26 @@ address. If you'd like to redirect payments to an account that is neither the co
 stash account, set one up. Note that setting an exchange address as the
 recipient of the staking rewards is extremely unsafe.
 
+## Nominating vs Joining a Pool
+
+Nominating is the action of choosing validators. It does not simply involve bonding tokens. Nominating is an active task, which implies that you regularly monitor that your stake is backing an active validator in all the eras and check if you are receiving your staking rewards. More importantly, ensure that the validators you chose always act in the best interests of the network protocol and have less chance of getting slashed. To nominate, you need a minimum of 10 DOT, and to receive rewards, you need at least a balance greater than the minimum active bond. Depending on your validators, if your active validator is oversubscribed, you will earn rewards only if your stake is within that of the top 256 nominators. If the validator misbehaves, It is worth noting that your stake is subject to slashing, irrespective of whether you are in the top 256 nominators or not.
+
+As the minimum active bond is a dynamic value, it can make your nomination inactive when the threshold goes above your bonded balance. Hence, to be eligible to earn rewards while nominating, you would need to stake a much higher balance than the minimum active bond.
+
+Nomination pools are a way to participate in staking with as little as 1 DOT and earn staking rewards. Nomination pools differ from custodial solutions (like staking through central exchanges) because they are non-custodial, native to Polkadot's protocol, permissionless, transparent, and run in a decentralized way by the community. Before joining a nomination pool, you must ensure that the pool is earning rewards and nominating the validators that match your preferences. Participating in pools is more of a set-and-forget action than nominating by yourself. It is worth noting that it is the pool operator that maintains the list of validators nominated by the pool, and so, in a way, you are trusting the pool operator to act in your best interests. However, it is advised to check the validators nominated by the pool from time to time and change the pool if necessary.
+
+| Nominating | Joining a Pool|
+|:--------:|:----------:|
+| Minimum 10 DOT to nominate. | Minimum 1 DOT to be a member. |
+| Unlimited bonding period. | Unlimited bonding period until the pool exists. |
+| Unbonding period of 28 days. | Unbonding period of 28 days. |
+| Maximum uncapped. | Maximum uncapped. |
+| Staking rewards not guaranteed. | Staking rewards guaranteed. |
```

Comment:
This is not true. A pool may not earn rewards when none of the validators it nominates make it to the active set
```suggestion
| Should bond more than the [minimum active nomination](../learn/learn-nominator.md#minimum-active-nomination-to-receive-staking-rewards) in an era to be eligible to earn staking rewards, although it can depend on multiple other factors outlined in the linked document. |  A nomination pool earns rewards in an era if it satisfies all the conditions mentioned for the nominator (as the nomination pool is just a nominator from [the NPoS system](../learn/learn-phragmen.md) perspective  |
```